bin/shoryuken

```diff
@@ -21,7 +21,7 @@ module Shoryuken
       method_option :daemon,      aliases: '-d', type: :boolean, desc: 'Daemonize process'
       method_option :queues,      aliases: '-q', type: :array,   desc: 'Queues to process with optional weights'
       method_option :require,     aliases: '-r', type: :string,  desc: 'Dir or path of the workers'
-      method_option :config_file, aliases: '-C', type: :string,  desc: 'Path to config file'
+      method_option :config,      aliases: '-C', type: :string,  desc: 'Path to config file'
```

Comment:
hm @fredsadaghiani that ^ should break [this](https://github.com/phstc/shoryuken/blob/60905beefc0c8d103aac6b0dd22ede7bb4f71d7f/lib/shoryuken/environment_loader.rb#L43), it expects `config_file`.

We would need to change it in both places.  

Or in the `start` body, create an alias `ops[:config_file] = options[:config]`.

Thinking more on it, if we change it now, it would break for people using version 3 already. Version 3 is a [major version with breaking changes](http://semver.org/).

Maybe one possibility would be to use [`aliases`](https://github.com/erikhuda/thor/wiki/Method-Options#available-options) to set `config` as an alias for `config_file`, WDYT?